08-building-search-applications/translations/ja-jp/README.md

```diff
@@ -0,0 +1,165 @@
+# 検索アプリケーションの構築
+
+[![Introduction to Generative AI and Large Language Models](../../images/08-lesson-banner.png?WT.mc_id=academic-105485-yoterada)](TBD)
+
+> **ビデオは近日公開予定**
+
+LLM は、チャットボットやテキスト生成だけでなく、Embeddings を使った検索アプリケーションの構築も可能です。Embeddings とは、ベクトルとも呼ばれるデータの数値表現で、データのセマンティック検索に活用できます。
+
+今回のレッスンでは、私たちの教育スタートアップ向けの検索アプリケーションを作ります。私たちのスタートアップは、発展途上国の学生に無料教育を提供する非営利団体で、AI について学べる多数の YouTube ビデオを提供しています。そして今、学生が質問を打ち込むだけで YouTube の動画を検索できるアプリケーションの開発を目指しています。
+
+例えば、学生が「Jupyter Notebookとは何か？」や「Azure MLとは何か？」と質問を入力すれば、その質問に関連するYouTube動画のリストを返します。さらに、その質問の答えが動画のどの部分にあるのかを示すリンクも提供します。
+
+## はじめに
+
+このレッスンでは、下記の内容を取り扱います：
+
+- セマンティック検索とキーワード検索の違い
+- テキスト埋め込みとは何か
+- テキスト埋め込みインデックスの作成方法
+- テキスト埋め込みインデックスの検索方法
+
+## 学習目標
+
+このレッスンを終了後、下記のことができるようになります：
+
+- セマンティック検索とキーワード検索の違いを理解し、説明できる
+- テキスト埋め込みとは何かを理解し、説明できる
+- データ検索に Embeddings を使用したアプリケーションを作成できる
+
+## なぜ検索アプリケーションを作るのでしょうか？
+
+実際に検索アプリケーションを作ることで、Embeddings を使ったデータ検索方法を理解することができます。また、学生が情報を素早く見つけるための検索アプリケーションの作り方も学べます。
+
+このレッスンでは、Microsoft の [AI Show](https://www.youtube.com/playlist?list=PLlrxD0HtieHi0mwteKBOfEeOYf0LJU4O1?WT.mc_id=academic-105485-yoterada) という YouTube チャンネルから、YouTube の字幕を抜き出し、それを Embedding 検索用のインデックスとして利用する方法について紹介します。AI Show は、AI と機械学習について学ぶことのできる YouTube チャンネルです。Embedding 用のインデックスには、2023 年 10 月までの各 YouTube 字幕の Embeddings が含まれています。この Embedding インデックスを使用して、スタートアップ用の検索アプリケーションを作ります。この検索アプリケーションは、質問の答えが動画のどの部分にあるかを示すリンクを返します。これは、学生が必要な情報を素早く見つけるのに非常に役立ちます。
+
+下記は、「Azure MLで rstudio を使うことはできますか？」という質問に対するセマンティック・クエリの例です。YouTube の URL を見てみると、URL には質問の答えがどの動画のどの部分にあるかを示すタイムスタンプが含まれています。
+
+![Semantic query for the question "can you use rstudio with Azure ML"](../../images/query-results.png?WT.mc_id=academic-105485-yoterada)
+
+## セマンティック検索とは何でしょうか？
+
+セマンティック検索とは、検索クエリ内の単語の意味、つまりセマンティクスを利用して、関連性のある結果を返す検索手法のことを指します。
+
+セマンティック検索の一例を挙げてみましょう。あなたが車を購入したいと考えていて、「私が将来乗りたい夢の車」を検索したとします。この場合、セマンティック検索は、あなたが車について「夢」見ているわけではなく、「理想の車」を購入したいという意図を理解します。そして、その意図に関連する結果を返します。これに対して、「キーワード検索」は文字通りに車についての夢を検索し、結果として関連性のない情報を返すことが多くなります。
+
+## テキスト埋め込み (Embeddings) とは何でしょうか？
+
+[テキストの埋め込み (Embeddings)](https://en.wikipedia.org/wiki/Word_embedding?WT.mc_id=academic-105485-yoterada) は、[自然言語処理](https://en.wikipedia.org/wiki/Natural_language_processing?WT.mc_id=academic-105485-yoterada)において用いられるテキストの表現方法で、テキストの意味を数値的に表現したものです。この埋め込みは、機械が理解しやすいように数値データとして表現するために使用します。テキスト埋め込みを生成するための AI モデルは多数存在しますが、今回のレッスンでは OpenAI が提供する埋め込みモデルを使用して埋め込みを生成する方法に焦点を当てます。
+
+Here's an example, imagine the following text is in a transcript from one of the episodes on the AI Show YouTube channel:
```

Comment:
```suggestion
```

You've already translated this part into Japanese at [line 52](https://github.com/microsoft/generative-ai-for-beginners/pull/230/files#diff-420250f0802e53e42481895d4d7a925e8c9806704711948e6cc717b0290e07b7R52) , so I think the original text here isn't necessary. 😄